src/network/http.py

```diff
@@ -19,13 +21,15 @@ def __init__(self, host, path="/"):
         print "connecting in background to %s:%i" % (self.destination[0], self.destination[1])
 
     def state_init(self):
-        self.append_write_buf("GET %s HTTP/1.1\r\nHost: %s\r\nConnection: close\r\n\r\n" % (self.path, self.destination[0]))
-        print "Sending %ib"  % (len(self.write_buf))
+        self.append_write_buf(
+            "GET %s HTTP/1.1\r\nHost: %s\r\nConnection: close\r\n\r\n" % (
+                self.path, self.destination[0]))
+        print "Sending %ib" % (len(self.write_buf))
         self.set_state("http_request_sent", 0)
         return False
 
     def state_http_request_sent(self):
-        if len(self.read_buf) > 0:
+        if len(self.read_buf) > 0:          # pylint: disable=len-as-condition
```

Comment:
this can probably be rewritten as `if self.read_buf`.

Reply:
Okay .